docs/contributing/developing.rst

```diff
@@ -116,6 +116,35 @@ If your Elasticsearch instance is located somewhere else, you can set the
 
     $ ELASTICSEARCH_URL=http://my-elasticsearch-instance:9200 python runtests.py --elasticsearch
 
+**Browser and device support**
+
+Wagtail is meant to be used on a wide variety of devices and browsers. Supported browser / device versions include:
+
+=============  =============  =============
+Browser        Device/OS      Version(s)
+=============  =============  =============
+Mobile Safari  iOS Phone      Latest
+Mobile Safari  iOS Tablet     Latest
+Chrome         Android        Latest
```

Comment:
Maybe we should refer to the last 2 versions instead of latest (for all)? 
That way we're are not really attached to the absolute latest but we also for eg. include iOS 9, which together with iOS 10 holds up to 94% (https://developer.apple.com/support/app-store/)